### ISSUE TITLE
Mapping multiple Bluetooth controllers to one in-game controller

### ISSUE THREAD

**Issue by AmNotOnline** [mupen64plus-ae/mupen64plus-ae#743]
Since I found out that you can use Nintendo Switch Joy-cons as Bluetooth controllers I've tried to connect them both to one in-game controller (because on Joy-con doesn't have enough buttons) 
I also have an app which is in place of the normal android keyboard. It works by mapping the buttons and then it sends out a signal (like KEYCODE_BUTTON_Z) and does support using multiple controllers at once,but as far as I know I can't use it for mapping the in-game controller.

TL;DR I can't hook up two real life Bluetooth controllers up to one in-game controller.

**Comment by fzurita** [comment]
Go to profiles then select profiles. Select the multiplayer option. In there, try assigning the two controllers to player 1.

**Comment by AmNotOnline** [comment]
Thanks a lot for the help, it works and I'm now capable of playing a game without buying an extra controller
